### ISSUE TITLE
训练topk一直为0

### ISSUE THREAD

**Issue by czzzy** [PaddlePaddle/PaddleClas#2709]
我按照
https://github.com/PaddlePaddle/PaddleClas/blob/release/2.5/docs/zh_CN/quick_start/quick_start_classification_new_user.md中的教程训练花的分类，训练输出如下
![image](https://user-images.githubusercontent.com/27075974/225521462-a285bed4-3d0f-4af1-9cac-31c53f2feff5.png)
参数配置如下
[2023/03/16 12:50:48] ppcls INFO: 
===========================================================
==        PaddleClas is powered by PaddlePaddle !        ==
===========================================================
==                                                       ==
==   For more info please go to the following website.   ==
==                                                       ==
==       https://github.com/PaddlePaddle/PaddleClas      ==
===========================================================

[2023/03/16 12:50:48] ppcls INFO: Arch : 
[2023/03/16 12:50:48] ppcls INFO:     class_num : 102
[2023/03/16 12:50:48] ppcls INFO:     name : ResNet50_vd
[2023/03/16 12:50:48] ppcls INFO: DataLoader :
[2023/03/16 12:50:48] ppcls INFO:     Eval :
[2023/03/16 12:50:48] ppcls INFO:         dataset :
[2023/03/16 12:50:48] ppcls INFO:             cls_label_path : ./dataset/flowers102/val_list.txt
[2023/03/16 12:50:48] ppcls INFO:             image_root : ./dataset/flowers102
[2023/03/16 12:50:48] ppcls INFO:             name : ImageNetDataset
[2023/03/16 12:50:48] ppcls INFO:             transform_ops :
[2023/03/16 12:50:48] ppcls INFO:                 DecodeImage :
[2023/03/16 12:50:48] ppcls INFO:                     channel_first : False
[2023/03/16 12:50:48] ppcls INFO:                     to_rgb : True
[2023/03/16 12:50:48] ppcls INFO:                 ResizeImage :
[2023/03/16 12:50:48] ppcls INFO:                     resize_short : 256
[2023/03/16 12:50:48] ppcls INFO:                 CropImage :
[2023/03/16 12:50:48] ppcls INFO:                     size : 224
[2023/03/16 12:50:48] ppcls INFO:                 NormalizeImage :
[2023/03/16 12:50:48] ppcls INFO:                     mean : [0.485, 0.456, 0.406]
[2023/03/16 12:50:48] ppcls INFO:                     order :
[2023/03/16 12:50:48] ppcls INFO:                     scale : 1.0/255.0
[2023/03/16 12:50:48] ppcls INFO:                     std : [0.229, 0.224, 0.225]
[2023/03/16 12:50:48] ppcls INFO:         loader :
[2023/03/16 12:50:48] ppcls INFO:             num_workers : 0
[2023/03/16 12:50:48] ppcls INFO:             use_shared_memory : True
[2023/03/16 12:50:48] ppcls INFO:         sampler :
[2023/03/16 12:50:48] ppcls INFO:             batch_size : 64
[2023/03/16 12:50:48] ppcls INFO:             drop_last : False
[2023/03/16 12:50:48] ppcls INFO:             name : DistributedBatchSampler
[2023/03/16 12:50:48] ppcls INFO:             shuffle : False
[2023/03/16 12:50:48] ppcls INFO:     Train :
[2023/03/16 12:50:48] ppcls INFO:         dataset :
[2023/03/16 12:50:48] ppcls INFO:             cls_label_path : ./dataset/flowers102/train_list.txt
[2023/03/16 12:50:48] ppcls INFO:             image_root : ./dataset/flowers102
[2023/03/16 12:50:48] ppcls INFO:             name : ImageNetDataset
[2023/03/16 12:50:48] ppcls INFO:             transform_ops :
[2023/03/16 12:50:48] ppcls INFO:                 DecodeImage :
[2023/03/16 12:50:48] ppcls INFO:                     channel_first : False
[2023/03/16 12:50:48] ppcls INFO:                     to_rgb : True
[2023/03/16 12:50:48] ppcls INFO:                 RandCropImage :
[2023/03/16 12:50:48] ppcls INFO:                     size : 224
[2023/03/16 12:50:48] ppcls INFO:                 RandFlipImage :
[2023/03/16 12:50:48] ppcls INFO:                     flip_code : 1
[2023/03/16 12:50:48] ppcls INFO:                 NormalizeImage :
[2023/03/16 12:50:48] ppcls INFO:                     mean : [0.485, 0.456, 0.406]
[2023/03/16 12:50:48] ppcls INFO:                     order :
[2023/03/16 12:50:48] ppcls INFO:                     scale : 1.0/255.0
[2023/03/16 12:50:48] ppcls INFO:                     std : [0.229, 0.224, 0.225]
[2023/03/16 12:50:48] ppcls INFO:         loader :
[2023/03/16 12:50:48] ppcls INFO:             num_workers : 8
[2023/03/16 12:50:48] ppcls INFO:             use_shared_memory : True
[2023/03/16 12:50:48] ppcls INFO:         sampler :
[2023/03/16 12:50:48] ppcls INFO:             batch_size : 16
[2023/03/16 12:50:48] ppcls INFO:             drop_last : False
[2023/03/16 12:50:48] ppcls INFO:             name : DistributedBatchSampler
[2023/03/16 12:50:48] ppcls INFO:             shuffle : True
[2023/03/16 12:50:48] ppcls INFO: Global :
[2023/03/16 12:50:48] ppcls INFO:     checkpoints : None
[2023/03/16 12:50:48] ppcls INFO:     device : gpu
[2023/03/16 12:50:48] ppcls INFO:     epochs : 20
[2023/03/16 12:50:48] ppcls INFO:     eval_during_train : True
[2023/03/16 12:50:48] ppcls INFO:     eval_interval : 1
[2023/03/16 12:50:48] ppcls INFO:     image_shape : [3, 224, 224]
[2023/03/16 12:50:48] ppcls INFO:     output_dir : ./output/
[2023/03/16 12:50:48] ppcls INFO:     pretrained_model : None
[2023/03/16 12:50:48] ppcls INFO:     print_batch_step : 10
[2023/03/16 12:50:48] ppcls INFO:     save_inference_dir : ./inference
[2023/03/16 12:50:48] ppcls INFO:     save_interval : 1
[2023/03/16 12:50:48] ppcls INFO:     use_visualdl : False
[2023/03/16 12:50:48] ppcls INFO: Infer :
[2023/03/16 12:50:48] ppcls INFO:     PostProcess :
[2023/03/16 12:50:48] ppcls INFO:         class_id_map_file : ./datase/flowers102t/labels.txt
[2023/03/16 12:50:48] ppcls INFO:         name : Topk
[2023/03/16 12:50:48] ppcls INFO:         topk : 5
[2023/03/16 12:50:48] ppcls INFO:     batch_size : 2
[2023/03/16 12:50:48] ppcls INFO:     infer_imgs : docs/images/inference_deployment/whl_demo.jpg
[2023/03/16 12:50:48] ppcls INFO:     transforms :
[2023/03/16 12:50:48] ppcls INFO:         DecodeImage :
[2023/03/16 12:50:48] ppcls INFO:             channel_first : False
[2023/03/16 12:50:48] ppcls INFO:             to_rgb : True
[2023/03/16 12:50:48] ppcls INFO:         ResizeImage :
[2023/03/16 12:50:48] ppcls INFO:             resize_short : 256
[2023/03/16 12:50:48] ppcls INFO:         CropImage :
[2023/03/16 12:50:48] ppcls INFO:             size : 224
[2023/03/16 12:50:48] ppcls INFO:         NormalizeImage :
[2023/03/16 12:50:48] ppcls INFO:             mean : [0.485, 0.456, 0.406]
[2023/03/16 12:50:48] ppcls INFO:             order :
[2023/03/16 12:50:48] ppcls INFO:             scale : 1.0/255.0
[2023/03/16 12:50:48] ppcls INFO:             std : [0.229, 0.224, 0.225]
[2023/03/16 12:50:48] ppcls INFO:         ToCHWImage : None
[2023/03/16 12:50:48] ppcls INFO: Loss :
[2023/03/16 12:50:48] ppcls INFO:     Eval : 
[2023/03/16 12:50:48] ppcls INFO:         CELoss :
[2023/03/16 12:50:48] ppcls INFO:             weight : 1.0
[2023/03/16 12:50:48] ppcls INFO:     Train :
[2023/03/16 12:50:48] ppcls INFO:         CELoss :
[2023/03/16 12:50:48] ppcls INFO:             weight : 1.0
[2023/03/16 12:50:48] ppcls INFO: Metric :
[2023/03/16 12:50:48] ppcls INFO:     Eval :
[2023/03/16 12:50:48] ppcls INFO:         TopkAcc :
[2023/03/16 12:50:48] ppcls INFO:             topk : [1, 5]
[2023/03/16 12:50:48] ppcls INFO:     Train :
[2023/03/16 12:50:48] ppcls INFO:         TopkAcc :
[2023/03/16 12:50:48] ppcls INFO:             topk : [1, 5]
[2023/03/16 12:50:48] ppcls INFO: Optimizer :
[2023/03/16 12:50:48] ppcls INFO:     lr :
[2023/03/16 12:50:48] ppcls INFO:         learning_rate : 0.0125
[2023/03/16 12:50:48] ppcls INFO:         name : Cosine
[2023/03/16 12:50:48] ppcls INFO:         warmup_epoch : 5
[2023/03/16 12:50:48] ppcls INFO:     momentum : 0.9
[2023/03/16 12:50:48] ppcls INFO:     name : Momentum
[2023/03/16 12:50:48] ppcls INFO:     regularizer :
[2023/03/16 12:50:48] ppcls INFO:         coeff : 1e-05
[2023/03/16 12:50:48] ppcls INFO:         name : L2
[2023/03/16 12:50:48] ppcls INFO: profiler_options : None
[2023/03/16 12:50:48] ppcls INFO: train with paddle 2.4.1 and device Place(gpu:0)

**Comment by cuicheng01** [comment]
麻烦问下这里有改过代码吗

**Comment by czzzy** [comment]
没有改过代码

**Comment by cuicheng01** [comment]
完全是使用的文档的教程跑的吗？
